core/modules/replica/ConfigApp.cc

```diff
@@ -96,12 +385,81 @@ ConfigApp::ConfigApp(int argc, char* argv[])
          "ADD_TABLE", "DELETE_TABLE"
         },
         _command
+    ).option(
+        "config",
+        "Configuration URL (a configuration file or a database connection string).",
```

Comment:
This is an artifact in the refactored code since it no longer supports configuration files. Only the database connection strings are now allowed. Fixed.